lib/internal/test_runner/tap_stream.js

```diff
@@ -127,8 +127,12 @@ class TapStream extends Readable {
 // In certain places, # and \ need to be escaped as \# and \\.
 function tapEscape(input) {
   return StringPrototypeReplaceAll(
-    StringPrototypeReplaceAll(input, '\\', '\\\\'), '#', '\\#'
-  );
+            StringPrototypeReplaceAll(
```

Comment:
Can you fix the linting issues?

Reply:
Running `make lint-js-fix` (or `vcbuild lint-js-fix` on Windows) should fix most of these lint issues automatically.